target/linux/bcm53xx/config-6.6

```diff
@@ -231,10 +242,13 @@ CONFIG_NEED_DMA_MAP_STATE=y
 CONFIG_NEED_SRCU_NMI_SAFE=y
 CONFIG_NET_DEVLINK=y
 CONFIG_NET_DSA=y
+CONFIG_NET_DSA_QCA8K=y
+CONFIG_NET_DSA_QCA8K_LEDS_SUPPORT=y
```

Comment:
When you compile in the switch driver it is compiled in for all boards using this target. Is it possible to use `kmod-dsa-qca8k` instead?

Reply:
I considered that, but it breaks failsafe mode, as the switch driver probes after preinit network setup in this case, when the module is loaded.

Maybe we should reorder probing boot modules and preinit network setup instead? This would work, but sounds more risky to me, at least for now.

Reply:
Another option would be to create a subtarget for Meraki devices. This would help me sort out the LED issue - currently status LED on MR26 and MR32 is inconsistent with the rest of Merakis. This way I could use generic `/etc/diag.sh` to handle them, and patch only Meraki device trees appropriately.

Reply:
...on the other hand - another subtarget is additional load for buildbots, because it means different kernel configuration, which is substantial when building with `CONFIG_ALL_KMODS` and `CONFIG_ALL_NONSHARED`, like we do in official images.
@hauke what do you think?

Reply:
I am against an additional subtarget because it adds a lot of extra maintenance efforts. If it does not work well when these are build as module it should be fine to add them to the kernel. If all other devices are nand anyway we should have enough space.

Reply:
Understood.